test/utils.js

```diff
@@ -89,64 +69,42 @@ module.exports.extractInfoFromResponse = function (res) {
     let idRefreshTokenHttpOnly = false;
     let refreshTokenHttpOnly = false;
 
+    let cookies = [];
     if (res instanceof CollectingResponse) {
-        const accessTokenCookie = res.cookies.find((info) => info.key === "sAccessToken");
-        if (accessTokenCookie) {
-            accessToken = accessTokenCookie.value;
-            accessTokenExpiry = new Date(accessTokenCookie.expires).toUTCString();
-            accessTokenDomain = accessTokenCookie.domain;
-            accessTokenHttpOnly = accessTokenCookie.httpOnly;
-        }
-        const refreshTokenCookie = res.cookies.find((info) => info.key === "sRefreshToken");
-        if (refreshTokenCookie) {
-            refreshToken = refreshTokenCookie.value;
-            refreshTokenExpiry = new Date(refreshTokenCookie.expires).toUTCString();
-            refreshTokenDomain = refreshTokenCookie.domain;
-            refreshTokenHttpOnly = refreshTokenCookie.httpOnly;
-        }
         headers = Object.fromEntries(res.headers.entries());
+        cookies = res.cookies;
     } else {
```

Comment:
```suggestion
    } else if(res) {
```
if res is undefined, the test will fail